src/test/resources/com/aws/greengrass/smbridge/config_with_mapping.yaml

```diff
@@ -1,12 +1,12 @@
 services:
-  aws.greengrass.MqttBridge:
+  aws.greengrass.smbridge:
     parameters:
       brokerServerUri: 'tcp://localhost:8883'
-      mqttTopicMapping: '[
+      mqttStreamMapping: '[
                            {"SourceTopic": "topic/to/iotCore", "SourceTopicType": "LocalMqtt", "DestTopic": "/test/cloud/topic", "DestTopicType": "IotCore"},
                            {"SourceTopic": "topic/to/pubsub", "SourceTopicType": "LocalMqtt", "DestTopic": "/test/pubsub/topic", "DestTopicType": "Pubsub"},
                            {"SourceTopic": "topic/to/iotCore/2", "SourceTopicType": "LocalMqtt", "DestTopic": "/test/cloud/topic2", "DestTopicType": "IotCore"}
                          ]'
   main:
     dependencies:
-      - aws.greengrass.MqttBridge
+      - aws.greengrass.smbridge
```

Comment:
Can you also add an example of the stream definition config?

Reply:
Configs aren't necessary for this package since we are doing integration testing in UATs (because JUnit doesn't handle the dependency on SM too well...). Do you think it is still a good idea to include example configs in-package? It's totally doable.

Reply:
That is fine. In that case, it will nothing but do out of sync.